elytron/src/main/java/org/wildfly/extension/elytron/RoleMapperDefinitions.java

```diff
@@ -80,6 +85,19 @@
         .setRestartAllServices()
         .build();
 
+    static final SimpleAttributeDefinition REGEX = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.REGEX, ModelType.STRING, false)
```

Comment:
I think you can use `SimpleAttributeDefinitionBuilder(RegexAttributeDefinitions.PATTERN)` here instead and then remove the `set` calls below.

Reply:
Fixed. Thank you